client/my-sites/stats/site.jsx

```diff
@@ -406,20 +408,28 @@ class StatsSite extends Component {
 					</div>
 				</div>
 
-				<div className="stats-content-promo">
-					<PromoCardBlock
-						productSlug="wordpress-seo-premium"
-						impressionEvent="calypso_stats_wordpress_seo_premium_banner_view"
-						clickEvent="calypso_stats_wordpress_seo_premium_banner_click"
-						headerText={ translate( 'Increase site visitors with Yoast SEO Premium' ) }
-						contentText={ translate(
-							'Purchase Yoast SEO Premium to ensure that more people find your incredible content.'
-						) }
-						ctaText={ translate( 'Learn more' ) }
-						image={ wordpressSeoIllustration }
-						href={ `/plugins/wordpress-seo-premium/${ slug }` }
-					/>
+				<div className="stats-traffic-promo-container">
+					<DotPager className="stats-traffic-promo-pager">
```

Comment:
```suggestion
					<DotPager className="stats__promo-pager">
```

As what is mentioned above for BEM.